spread.yaml

```diff
@@ -448,7 +457,7 @@ suites:
         # Test cases are not yet ported to Fedora/openSUSE that is why
         # we keep them disabled. A later PR will enable most tests and
         # drop this blacklist.
-        systems: [-ubuntu-core-16-*, -fedora-*, -opensuse-*]
+        systems: [-ubuntu-core-16-*, -fedora-*, -opensuse-*, -arch-*]
```

Comment:
Could you please add a line explaining why arch is skiped

Reply:
Updated the comment. I think it should be possible to enable `tests/unit/..` on Fedora, openSUSE and Arch in a followup PR.